app/consumer-democracy/proposals_whitelisting_test.go

```diff
@@ -12,7 +12,8 @@ import (
 )
 
 func TestDemocracyGovernanceWhitelistingKeys(t *testing.T) {
-	_, valUpdates, _ := testutil.CreateValidators(t, 4)
+	_, valUpdates, _, err := testutil.CreateValidators(4)
```

Comment:
I changed the signature of the CreateValidators function to better suit my needs, so I had to adjust this here